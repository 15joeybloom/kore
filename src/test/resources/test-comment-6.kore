[]

module TEST-COMMENT-6

  sort{} Nat{ /* Block comments are whitespaces. */ } [] 

endmodule []

