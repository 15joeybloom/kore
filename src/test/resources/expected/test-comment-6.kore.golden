Definition {definitionAttributes = Attributes {getAttributes = []}, definitionModules = Module {moduleName = ModuleName {getModuleName = "TEST-COMMENT-6"}, moduleSentences = [SentenceSortSentence (SentenceSort {sentenceSortName = Id {getId = "Nat"}, sentenceSortParameters = [], sentenceSortAttributes = Attributes {getAttributes = []}})], moduleAttributes = Attributes {getAttributes = []}}}